src/components/Button/Button.js

```diff
@@ -33,17 +31,50 @@ const Button = ({ className, largeText, size, ...other }) => {
   );
 };
 
+/* eslint-disable react/require-default-props */
 Button.defaultProps = {
-  ...defaultProps,
   largeText: null,
+  icon: 'None',
+  kind: 'primary',
+  disabled: false,
+  size: 'default',
 };
 
 Button.propTypes = {
-  ...propTypes,
+  /**
```

Comment:
- [ ] As we build on top of Carbon's `Button`, these `propTypes` should reflect Carbon's, with the exception of `largeText` and `size` (as we introduce a `large` variant), which should remain as-is - https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Button/Button.js#L105